METRICS_CONFIGURATION.md

```
@@ -0,0 +1,313 @@
+   metrics:
+     endpoint: "/internal/prometheus" # Harder to guess
+   ```
+
+2. **Use network-level security:**
+
+   - Configure firewall rules to allow only Prometheus server
+   - Use private network for metrics collection
+   - Deploy Prometheus in the same VPC/network
+
+3. **Reverse proxy with authentication:**
+   ```nginx
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)